mne/surface.py

```diff
@@ -32,15 +32,15 @@
 # BEM
 
 @verbose
-def read_bem_surfaces(fname, add_geom=False, s_id=None, verbose=None):
+def read_bem_surfaces(fname, patch_stats=False, s_id=None, verbose=None):
```

Comment:
here for example would it be:
``def read_bem_surfaces(fname, patch_stats=False, add_geom=None, s_id=None, verbose=None):``

and then the check you had.